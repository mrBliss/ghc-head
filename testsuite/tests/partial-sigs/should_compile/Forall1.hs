{-# LANGUAGE PartialTypeSignatures, ScopedTypeVariables #-}
module Forall1 where

fall :: forall a. _ -> a
fall v = v

test :: (Char, Bool)
test = (fall 'c', fall True)
