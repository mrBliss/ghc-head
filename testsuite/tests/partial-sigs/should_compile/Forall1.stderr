TYPE SIGNATURES
  fall :: forall a. a -> a
TYPE CONSTRUCTORS
COERCION AXIOMS
Dependent modules: []
Dependent packages: [base, ghc-prim, integer-gmp]
